x-pack/plugins/enterprise_search/public/applications/shared/truncate/truncated_content.scss

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+.truncated-content {
+  position: relative;
+  z-index: 2;
+  display: inline-block;
+  white-space: nowrap;
+
+  &__tooltip {
+    @include position('vertical');
```

Comment:
Do we have this mixin in Kibana?

Reply:
Yet another great catch. I didn't see one. I just pulled out the relevant styles for now since it only adds 2 lines